### ISSUE TITLE
Q. Export CSV des étudiants qui ont importé la saé 

### ISSUE THREAD

**Issue by AdrienPrugnaud** [avenirs-esr/karuta-APC#18]
Sortir un CSV des étudiants qui ont importé la saé fonction de m ci-dessus / ou sortir la liste de tous les étudiants d’un enseignant (portfolios étu qui lui sont partagés) avec tout ce qu’ils ont importés => travail de design => adresse mail étudiant, nom, prénom, formations ajoutées, saé importées, ???? pas 1.3

**Comment by jgribonvald** [comment]
même problématique que #16 et #17

avis e-portfolium @olivier-gerbe

**Comment by olivier-gerbe** [comment]
oui, c'est le même problème. seul un admin pourrait accéderai un rapport aux portfolios étudiants pour savoir ce qui a été fait.
